source/documentation/user-journeys.html.md.erb

```diff
@@ -9,73 +9,99 @@ weight: 40
 This API is in Alpha and may have breaking changes.
 </div>
 
-## Creating a TGP record
+## Create TGP record
 
-### Adding a goods record to Trader Goods Profile
+Traders can use your software to add new goods records to their Trader Goods Profile using the [Create a goods record](https://developer.qa.tax.service.gov.uk/api-documentation/docs/api/service/trader-goods-profiles/1.0/oas/page#tag/Goods-records/operation/createTraderGoodsProfileRecord) endpoint.
```

Comment:
This should point to https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/trader-goods-profiles/1.0/oas/page#tag/Goods-records/operation/createTraderGoodsProfileRecord
note the lack of "qa.tax" between "developer" and "service"